commands/helper_test.go

```diff
@@ -0,0 +1,31 @@
+package commands
+
+import (
+	"os"
+	"testing"
+)
+
+func TestGetDate(t testing.T) {
```

Comment:
The test suite is failing because the type of `t` here is `testing.T` instead of `*testing.T`